########## Testcase 0: Failed ##########
Expected Output: 
76.8
6
Your Output: 
76.8
6
11
----------------------------------------
########## Testcase 1: Failed ##########
Expected Output: 
5.5
1
Your Output: 
5.5
1
Traceback (most recent call last):
  File "week2.py", line 71, in <module>
    main()
  File "week2.py", line 39, in main
    depth(data)
  File "week2.py", line 5, in depth
    for i in each:
TypeError: 'int' object is not iterable
----------------------------------------
########## Testcase 2: Failed ##########
Expected Output: 
1
6
Your Output: 
1
6
1
----------------------------------------
########## Testcase 3: Failed ##########
Expected Output: 
5
4
Your Output: 
5
4
Traceback (most recent call last):
  File "week2.py", line 71, in <module>
    main()
  File "week2.py", line 39, in main
    depth(data)
  File "week2.py", line 5, in depth
    for i in each:
TypeError: 'int' object is not iterable
----------------------------------------
########## Testcase 4: Failed ##########
Expected Output: 
0
8
Your Output: 
0
8
1
----------------------------------------
########## Testcase 5: Failed ##########
Expected Output: 
5654134.9
10
Your Output: 
5654134.9
10
Traceback (most recent call last):
  File "week2.py", line 71, in <module>
    main()
  File "week2.py", line 39, in main
    depth(data)
  File "week2.py", line 5, in depth
    for i in each:
TypeError: 'int' object is not iterable
----------------------------------------
